extensions/objects/models/objects.py

```diff
@@ -309,12 +309,34 @@ class SetOfNormalizedString(BaseObject):
     }
 
 
-class MathLatexString(BaseObject):
-    """Math LaTeX string class."""
+class MathExpressionContent(BaseObject):
+    """Math Expression Content class."""
 
-    description = 'A LaTeX string.'
+    description = ('Math Expression Content dict.This is in form of a dict '
+                   'having two properties, raw_latex and svg_filename. Both '
```

Comment:
I would suggest having same description in different places.

Reply:
done